src/umpire/ResourceManager.cpp

```diff
@@ -163,6 +177,18 @@ ResourceManager::initialize()
     m_allocators.emplace_front(std::move(host_allocator));
   }
 
+  {
+    std::unique_ptr<strategy::AllocationStrategy>
+      allocator{
+        //util::wrap_allocator<strategy::AllocationTracker>(
+          registry.makeMemoryResource("NULL", getNextId())};
```

Comment:
Might want to change the name of this to something like `"__umpire_internal_null"`, and figure out how to hide internal resources from users.